src/id_generator.py

```diff
@@ -0,0 +1,7 @@
+class IDGenerator:
```

Comment:
Please align to PEP 8 naming convention!

Reply:
Do you mean because of the upper case "D"?
![grafik](https://github.com/user-attachments/assets/45be943d-7ea4-4388-9199-5207699fc5cf)

Reply:
Indeedestens

Reply:
It appears that since ID is a acronym, it should be upper case according to PEP 8.
(Didn't know that before looking it up either)

Reply:
(8 Points for Griffindor)